### PR TITLE
Adding the DB schema for enterprise into new version file. Updated HEAD

### PR DESCRIPTION
**Pull request by rajendramachani**
@ashutosh-mishra

### CODE REVIEW

File: gbpservice/neutron/db/migration/alembic_migrations/versions/d2aab79622fe_nfp_enterprise_db.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+#
+
+# revision identifiers, used by Alembic.
+revision = 'd2aab79622fe'
+down_revision = 'c1aab79622fe'
```
Comment (ashutosh-mishra) [inline gbpservice/neutron/db/migration/alembic_migrations/versions/d2aab79622fe_nfp_enterprise_db.py]: This is a hard-coded value, it wont be same always, it'll get change with gbp opensource code.
We need to maintain this somehow or whenever we are shifting from advanced to enterprise, always get the latest head, and fill that value as down revision.

File: gbpservice/neutron/db/migration/alembic_migrations/versions/d2aab79622fe_nfp_enterprise_db.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+                                ondelete='SET NULL'),
+        sa.ForeignKeyConstraint(['network_function_device_id'],
+                                ['nfp_network_function_devices.id'],
+                                ondelete='SET NULL'),
+        sa.PrimaryKeyConstraint('id')
+    )
+
+
+def downgrade():
+    pass
```
Comment (ashutosh-mishra) [inline gbpservice/neutron/db/migration/alembic_migrations/versions/d2aab79622fe_nfp_enterprise_db.py]: Are we supporting the case moving from enterprise to base/advanced, in that case we need this downgrade to remove enterprise table. Also we need to update head properly.

Comment (njagadish) [inline gbpservice/neutron/db/migration/alembic_migrations/versions/d2aab79622fe_nfp_enterprise_db.py]: Currently, this support is not available. But, we can consider for future.

File: gbpservice/neutron/db/migration/alembic_migrations/versions/d2aab79622fe_nfp_enterprise_db.py (hunk 1 above)
Comment (njagadish) [inline gbpservice/neutron/db/migration/alembic_migrations/versions/d2aab79622fe_nfp_enterprise_db.py]: Yes. A sample value was added to test. We need to check how to come up with this ID.

**Pull request merged**